src/Psy/functions.php

```
@@ -155,6 +157,16 @@ function bin()
                 $config['configFile'] = $configFile;
             }
 
+            // Handle --color
+            if ($input->getOption('color')) {
+                $config['colorMode'] = ColorMode::getForced();
+            }
+
+            // Handle --no-color
+            if ($input->getOption('no-color')) {
```

Let's throw an error if both --color and --no-color are specified.